app/controllers/api/v1/transactions_controller.rb

```diff
@@ -18,6 +18,17 @@ class TransactionsController < BaseOrganizationController
       end
     end
 
+    def_param_group :transfer do
+      param :transfer, Hash, required: true, action_aware: true do
+        param :amount, Integer, 'Amount', required: true
```

Comment:
@yunixon I guess `amount` is not integer. It should be `amount_cents` or it should be not integer